### DISCUSSION TITLE
Welcome to gee_tools Discussions!

### DISCUSSION THREAD

**Discussion by 12rambau** [gee-community/gee_tools#120] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by fitoprincipe** [comment]
This is awesome @12rambau, thank you! 😃

**Comment by acwatt** [comment]
Hi, I'm Aaron Watt. I'm working on Earth Engine javascript and python code as part of a larger project assessing the value of satellite data. Our project focuses estimating the potential value of satellite data used in federal regulation of air pollution in the united states. My general workflow is:

- Load a bunch of datasets
- reproject the datasets to conform to the same resolution
- aggregate all datasets to the daily level
- merge all datasets as bands in daily images of an image collection
- sample the image collection at location-days where we have ground truth data
- run a regression to estimate the error of one of the satellite-based datasets
- predict the error over all days and geography for use in further analysis of the value of satellite data

I am also becoming an EE trainer in my department, helping other economists use EE and understand where the cool resources are (like this repo).

**Reply by 12rambau** [reply]
That's super interesting welcome in the GEE community I guess and thanks placing us among the "cool resources" 😎 !
